.changeset/remove-base-clipboard-copy.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
```Suggestion
--- 
```
Ignore suggestion, blocking comment for approval.

**Don't merge until non-breaking changes are released in 2.5.0**